package.json

```diff
@@ -80,7 +81,8 @@
     "reset": "pnpm clean:all; rm pnpm-lock.yaml; pnpm symlink:root; pnpm install;",
     "run:prettier": "prettier --loglevel warn --ignore-path .eslintignore \"**/*.{js,jsx,ts,tsx,md,mdx,json,html,css,yml,yaml,graphql}\"",
     "size": "size-limit",
-    "storybook": "manypkg run examples/storybook-react start",
+    "storybook": "run-p fix:css:watch storybook:start",
```

Comment:
highlight for reviewers: Since we don't use emotion in storybook anymore, we have to run `pnpm fix:css` everytime when we change the theme. I made `pnpm storybook` to generate style files automagically when needed.